librecores-ci-openrisc/test-scripts/yosys.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+cd /src
+
+fusesoc library add mor1kx /src
```

Comment:
Do we need to hardcode mor1kx in the script? I am not sure whether it is applicable to other projects

Reply:
Yes it won't be applicable to other projects, but I did this according to discussions on PR https://github.com/openrisc/mor1kx/pull/87 , should I move it in mor1kx repository ?

Reply:
Done